packages/app-accounts/src/ChangePasswordButton.tsx

```diff
@@ -0,0 +1,261 @@
+// Copyright 2017-2018 @polkadot/app-accounts authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import { BareProps, FormErrorProps, I18nProps } from '@polkadot/ui-app/types';
+
+import React from 'react';
+import classes from '@polkadot/ui-app/util/classes';
+import Button from '@polkadot/ui-app/Button';
+import Error from '@polkadot/ui-app/Error';
+import keyring from '@polkadot/ui-keyring/index';
+
+import ChangePasswordModal from './ChangePasswordModal';
+import translate from './translate';
+
+type State = {
+  address: string,
+  error?: FormErrorProps,
```

Comment:
I really don't get this new `FormErrorProps` that is now in here. So if I look at the definition, we have this -

```
type FormErrorProps = React.ReactNode & {
  props: React.ReactNode
};
```

So basically the above State become when expanded -

```
type State = {
  ...
  props: {
   ...
  }
}
```

Could we please re-look at this. It really just seems to add unneeded complexity for no benefit. Let's rather stick to the basics making things less magical and maintainable.